web-local/src/lib/svelte-query/invalidation.ts

```diff
@@ -73,6 +73,14 @@ export const invalidateMetricsViewData = (
   });
 };
 
+export function invalidationForMetricsViewData(query, metricsViewName: string) {
+  const r = new RegExp(
```

Comment:
We should have a single method for matching metrics view query. Perhaps refactor invalidateMetricsViewData to call this method as well?